config/i18n-tasks.yml

```diff
@@ -106,16 +106,34 @@ ignore_missing:
     - 'deed.deed.*'
     - 'article.graph.title'
     - 'thredded.*'
-  es: &NOT_EN
```

Comment:
Not supported anymore. This seems to be a security vulnerability that i18n have patched long ago.